css/_variables.scss

```diff
@@ -28,3 +28,7 @@ $c__grey--blue: #607D8B;
 $c__black: #333333;
 $c__white: #ffffff;
 
+// Breakpoints
+$b__breakpoint--large: 1024px;
```

Comment:
Since breakpoint is related to sizing, you should prefix with `s` instead `$s__breakpoint--large`.